Eos/Soave-Redlich-Kwong/EOS.cpp

```diff
@@ -2,14 +2,14 @@
 
 namespace EOS {
 
-  bool initialized=false;
+  AMREX_GPU_DEVICE_MANAGED bool initialized=false;
   
-  amrex::Real Tc[NUM_SPECIES];
-  amrex::Real Bi[NUM_SPECIES];
-  amrex::Real oneOverTc[NUM_SPECIES];
-  amrex::Real sqrtOneOverTc[NUM_SPECIES];
-  amrex::Real sqrtAsti[NUM_SPECIES];
-  amrex::Real Fomega[NUM_SPECIES];
+  AMREX_GPU_DEVICE_MANAGED amrex::Real Tc[NUM_SPECIES];
+  AMREX_GPU_DEVICE_MANAGED amrex::Real Bi[NUM_SPECIES];
+  AMREX_GPU_DEVICE_MANAGED amrex::Real oneOverTc[NUM_SPECIES];
+  AMREX_GPU_DEVICE_MANAGED amrex::Real sqrtOneOverTc[NUM_SPECIES];
+  AMREX_GPU_DEVICE_MANAGED amrex::Real sqrtAsti[NUM_SPECIES];
+  AMREX_GPU_DEVICE_MANAGED amrex::Real Fomega[NUM_SPECIES];
```

Comment:
@jrood-nrel Is this consistent with DPC++/HIP requirements?

Reply:
No we can't use these.

Reply:
Best to follow this because I'm going to implement the explicit struct copies in PeleC when I get a chance https://github.com/AMReX-Codes/amrex/pull/1780.

Reply:
Yeah this is just a stop gap (makes it equivalent to what is currently in the code for gamma law), per our conversation over [here](https://github.com/AMReX-Combustion/PelePhysics/pull/139), changes will be needed to make this work for HIP/DPC++.